recipes/blast/build.sh

```diff
@@ -2,25 +2,39 @@
 
 set -e -x -o pipefail
 
-mkdir -p $PREFIX/bin
-
-if [ "$(uname)" == "Darwin" ]; then
-    echo "Platform: Mac"
-
-    cp $SRC_DIR/bin/* $PREFIX/bin/
-
-elif [ "$(expr substr $(uname -s) 1 5)" == "Linux" ]; then
-    echo "Platform: Linux"
-
-    cd $SRC_DIR/c++/
-    # --with-hard-runpath is needed otherwise BLAST programs would search
-    # libraries first in the directories defined by the LD_LIBRARY_PATH
-    # environment variable, instead of using the rpath specified by conda
-    ./configure --prefix=$PREFIX --with-hard-runpath
-
-    make -j${CPU_COUNT}
-
-    cp $SRC_DIR/c++/ReleaseMT/bin/* $PREFIX/bin/
-fi
+cd $SRC_DIR/c++/
+
+# --with-hard-runpath is needed otherwise BLAST programs would search
+# libraries first in the directories defined by the LD_LIBRARY_PATH
+# environment variable, instead of using the rpath specified by conda
+
+./configure \
+    --with-dll \
+    --with-mt \
+    --without-autodep \
+    --without-makefile-auto-update \
+    --with-flat-makefile \
+    --without-caution \
+    --without-dbapi \
+    --without-lzo \
+    --with-hard-runpath \
+    --without-debug \
+    --with-strip \
+    --without-downloaded-vdb \
```

Comment:
`--without-vdb` instead of `--without-downloaded-vdb`

Reply:
thx